teamwork/south_migrations/0001_initial.py

```diff
@@ -0,0 +1,192 @@
+# -*- coding: utf-8 -*-
```

Comment:
Since this moved from `migrations` to `south_migrations`, is there a config change that needs to be done to keep this working with Django 1.6?